addons/account/models/account_move.py

```diff
@@ -4152,7 +4152,7 @@ def button_draft(self):
             move.mapped('line_ids.analytic_line_ids').unlink()
 
         self.mapped('line_ids').remove_move_reconcile()
-        self.write({'state': 'draft', 'is_move_sent': False})
+        self.write({'state': 'draft'})
```

Comment:
```suggestion
        self.state = 'draft'
```
Since there's only one field left :p